third_party/txt/src/txt/platform_linux.cc

```diff
@@ -13,6 +13,9 @@
 namespace txt {
 
 std::vector<std::string> GetDefaultFontFamilies() {
+#ifdef FLUTTER_USE_FONTCONFIG
+  return {"TizenDefaultFont"};
+#else
```

Comment:
Do we need this change (caf01dd6f3422bbdd5b4dae3a441f0d4401caad5) if we're not to re-enable fontconfig?

Reply:
No, ``TizenDefaultFont`` is not an actual font. That's an alias in fontconfig

Reply:
Then you might revert the commit.

Reply:
I'm sorry but I don't get the point. Because..?

Reply:
**The** commit in this context is caf01dd6f3422bbdd5b4dae3a441f0d4401caad5.

Reply:
I'm still not sure if I understood your intentions... 

This PR is base on ``fontconfig``.  And PR that uses font config again and revert unnecessary our modification will be prepared another PR (Perhaps @bbrto21 Am I right boram? :smile: ).
The reason for doing that is because it would be good to track the related ``fontconfig`` in the future.(Just my opinion)

I'm sorry for the lack of explanation of this.

Reply:
https://github.com/flutter-tizen/engine/pull/106#pullrequestreview-681361371 by @bbrto21:

> I think you need to re-enable fontconfig in azure-pipelines.

https://github.com/flutter-tizen/engine/pull/106#issuecomment-859266980 by @bwikbs:

> Reverting existing code and applying fontconfig seem to be a little different topic, so I would like to talk about it in another PR.

If this PR depends on fontconfig, why not re-enable the fontconfig option in azure-pipelines.yml? The change inside the `#ifdef FLUTTER_USE_FONTCONFIG` block doesn't affect anything without the option enabled, so I thought you could safely revert the change. I'm sorry but I still don't get your point.

Reply:
@swift-kim Well... You're right. I just wanted to collect the related re-enable font config contents in one PR. 
There is no great reason...(To be honest, I didn't want to revert and clean-up the existing ones. And because it seems that @bbrto21 has already done that work, I was going to ask for him)
I didn't think much about it.. If it's too weird, I'll enable the font config right away.

Reply:
Okay, if there will be a follow-up PR, you may just briefly mention what changes will be made in that PR and how `TizenDefaultFont` is related to those changes.

Reply:
I've updated PR description!